src/type/validate.js

```diff
@@ -190,6 +202,15 @@ function validateDirectives(context: SchemaValidationContext): void {
           ],
         );
       }
+
+      if (arg.defaultValue !== undefined) {
+        validateDefaultValue(arg.defaultValue, arg.type).forEach((error) => {
+          context.reportError(
+            `Argument @${directive.name}(${arg.name}:) has invalid default value: ${error}`,
+            arg.astNode?.defaultValue,
```

Comment:
meta: these codecov comments are actually pretty helpful to strategize where to start with a test suite